CONTRIBUTING.md

```diff
@@ -362,4 +362,4 @@ The rules:
 If you are having trouble getting into the mood of idiomatic Go, we recommend
 reading through [Effective Go](https://golang.org/doc/effective_go.html). The
 [Go Blog](https://blog.golang.org) is also a great resource. Drinking the
-kool-aid is a lot easier than going thirsty.
```

Comment:
not sure why this appears to be changed.

Reply:
perhaps a change in newline 🤔